pkg/operator/rgw/controller.go

```diff
@@ -101,3 +106,35 @@ func (c *ObjectStoreController) onDelete(obj interface{}) {
 		logger.Errorf("failed to delete object store %s. %+v", objectStore.Name, err)
 	}
 }
+
+func storeChanged(oldStore, newStore ObjectStoreSpec) bool {
+	if oldStore.DataPool.Replicated.Size != newStore.DataPool.Replicated.Size {
```

Comment:
same thing here. Is there anything here that the users should not be changing?

Reply:
same response here :)